book/http.md

```diff
@@ -936,7 +939,7 @@ make it so that, if your browser is started without a URL being given,
 some specific file on your computer is opened. You can use that file
 for quick testing.
 
-1-3 *`data:` scheme*. Yet another scheme is `data`, which allows inlining
+1-3 *`data`*. Yet another scheme is `data`, which allows inlining
```

Comment:
If we want the colon and the word "scheme" we need to update `view-source` below, but I think it's OK without the "scheme" and there's value in keeping exercise names short.